project/MicroServiceBuild.scala

```diff
@@ -15,8 +15,8 @@ object MicroServiceBuild extends Build with MicroService {
 
 private object AppDependencies {
   import play.sbt.PlayImport._
-  private val pegdownVersion = "1.4.2"
-  private val scalatestVersion = "2.2.2"
+  private val pegdownVersion = "1.6.0"
```

Comment:
Have we spoke already about why these have changed? Are they relevant to the PR?

Reply:
To sync with Tai-Frontend